.gitignore

```diff
@@ -37,4 +37,9 @@ coverage/
 build/
 
 # package-lock.json
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+
+explanation.md
```

Comment:
Seems like local specific files are added to the .gitignore. Do we need those for every user of the project?

Reply:
This is a file I created for myself only. It has been removed from the repo and `.gitignore` respectively.